tests/core/test_flyer.py

```diff
@@ -359,6 +393,24 @@ async def test_read_configuration():
             "Input should be 'internal', 'edge_trigger', 'constant_gate' or "
             "'variable_gate' [type=enum, input_value='not_in_enum', input_type=str]",
```

Comment:
Make your new `error_msg` be specific like this please.

Reply:
This is the error message I get by default from pydantic 
`"1 validation error for TriggerInfo\nnumber_of_triggers\n  Field required [type=missing, input_value={'number': -100}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing"`

Did you mean I should create a custom error message using this https://docs.pydantic.dev/latest/errors/errors/#customize-error-messages

Reply:
I have made the changes ..Thank you :smile: